pkg/core/resources/model/resource.go

```diff
@@ -10,6 +10,9 @@ import (
 
 const (
 	DefaultMesh = "default"
+	// NoMesh defines a marker that resource is not bound to a Mesh.
+	// Resources not bound to a mesh (ScopeGlobal) should have an empty string in Mesh field.
+	NoMesh = ""
```

Comment:
hmm, can we use some special character/string instead? Not sure if "" is the universal way to designate NoMesh. How is this encoded in YAML/JSON?

Reply:
"Not sure if "" is the universal way to designate NoMesh " Hm, why not?

When string is empty, mesh field is excluded from the JSON/YAML.